newsrevenuehub/rev-engine
# [NRH-196] logs error and skips slack message if no integration

#### What's this PR do?
Logs at info level if no Slack integration exists for target, instead of crashing

#### How should this be manually tested?
Make a contribution for an Org without slack integrated. Observe a lack of explosions. Possible view logs for .info level message.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?

#### Before it can be merged
- [x] Need to get unit test coverage up just a skosh
